src/main/java/fact/features/video/CenterOfGravity.java

```diff
@@ -1,301 +0,0 @@
-package fact.features.video;
-
-import fact.Utils;
-import fact.hexmap.FactPixelMapping;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import stream.Data;
-import stream.Processor;
-import stream.annotations.Parameter;
-
-
-public class CenterOfGravity implements Processor
-{
-    
-    FactPixelMapping pixelMap = FactPixelMapping.getInstance();
-    
-    private int sliceCount;
-    
-    /**
-     * This function calculates the center of gravity for every slice. It uses only shower pixel.
-     * It also calculates the variance and covariance of the center of gravity.
-     */
```

Comment:
yes, we should have a look in his thesis if it was promissing, check its runtime and then decide to add it the stdAnalysis or delete it.